app/routes.py

```diff
@@ -0,0 +1,79 @@
+from flask import Flask
```

Comment:
I think it's advisable to put all modules that belong to the same class together and just separate with coma